src/osd/ECBackend.cc

```diff
@@ -1686,6 +1686,29 @@ int ECBackend::get_min_avail_to_read_shards(
   return 0;
 }
 
+void ECBackend::get_min_want_to_read_shards(
```

Comment:
This duplicates some of the logic in get_want_to_read_shards (without the overload).  I'd like to see the other overload invoke this.